tests/common/system_utils/docker.py

```diff
@@ -119,6 +119,7 @@ def swap_syncd(dut, creds):
     docker_syncd_name = "docker-syncd-{}".format(vendor_id)
     docker_rpc_image = docker_syncd_name + "-rpc"
 
+    dut.command("config bgp shutdown all")  # Force image download to go through mgmt network
```

Comment:
After this command is issued, it still needs some time for DUT to purge all the learnt IP routes. There is possible race condition here.

Reply:
Thanks, added a ready check!